axlearn/common/evaler.py

```diff
@@ -249,6 +249,16 @@ def _call_model(
             is_training=False,
         )
 
+    def _dispatch_global_batch(self, input_batch: NestedTensor) -> NestedTensor:
+        module = self.parent
+        while module is not None:
+            if isinstance(module, SpmdEvaler):
+                break
```

Comment:
nit -- you could also fold the break into the while condition.

Reply:
Done.